Formula/calicoctl.rb

```diff
@@ -0,0 +1,27 @@
+class Calicoctl < Formula
+  desc "Calico CLI tool"
+  homepage "https://www.projectcalico.org"
+  url "https://github.com/projectcalico/calicoctl.git",
+    :tag => "v1.6.1",
```

Comment:
Let's nudge this line and the one below it up another two spaces. It's an unusual indent but _generally_ the way these seem to be preferred.